ansible/inventory/group_vars/all/default.yml

```diff
@@ -0,0 +1,17 @@
+---
+hostname: ag
+docker_build: false
+
+autograder_user: autograder
+autograder_group: autograder
+autograder_home: /home/autograder
+
+quickfeed_root: /home/autograder/quickfeed
+quickfeed_ui_root: /home/autograder/quickfeed/public
+quickfeed_bin: /home/autograder/go/bin
+quickfeed_repo: https://github.com/r0qs/quickfeed.git
+quickfeed_branch: dev-tooling
+
+# Define the real values in the production config that should never be committed.
```

Comment:
Should these be defined in this file, or only as env vars?

Reply:
These are the default values for all group of hosts (dev or prod), and they can be overwritten in run time. But The values are wrong. It should point the to autograde organization and master branch.

Reply:
I think I made a mistake when commenting; I clicked on line 16, referring to these:
```ansible
quickfeed_github_key: <ghkey>
quickfeed_github_secret: <ghsecret>
```
I guess these can be in said file, as long as we don't commit it.

Reply:
Yes, we can also define in the `inventory/hosts` or pass it as parameters when deploying. I just declare them because they were used here when running the daemon: https://github.com/autograde/quickfeed/blob/b71aec1f1123f277596c35a9cb789aa8ad58b1ce/ansible/roles/quickfeed/templates/systemd/quickfeed.service.d/agenv.conf.j2